build/Branding.props

```diff
@@ -1,5 +1,6 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <BuildName>CLI</BuildName>
```

Comment:
So far I've been matching these to the name of the github repo (so lowercase `cli` here), but it is up to you. I don't know whether anyone will want to do case-sensitive matching of manifest xml name to repo. (The GitHub APIs do work case-insensitively when it comes to repo name.)

Reply:
Lower-case is fine by me...changed.